apps/admin-web/src/App.tsx

```diff
@@ -1,19 +1,95 @@
-import * as React from "react";
-import { BrowserRouter, Route, Switch } from "react-router-dom";
+import React, { useEffect } from "react";
+import { BrowserRouter, Link, Route, Switch } from "react-router-dom";
 
 import { ToolList } from "./components/pages";
 import { Events } from "./components/pages/tools";
+import { config } from "./config";
 import "./static/css/App.css";
 
-localStorage.debug = "warden:*";
+import { useAuth0 } from "./utils/react-auth0-wrapper";
+
+import { Layout, Menu, PageHeader, Spin } from "antd";
+
+const { Header, Content, Footer, Sider } = Layout;
+
+const MainContent: React.SFC<{}> = (): JSX.Element => {
+  return (
+    <Switch>
+      <Route exact={true} path="/" component={ToolList} />
+      <Route exact={true} path="/events" component={Events} />
+    </Switch>
+  );
+};
 
 const App: React.SFC<{}> = (): JSX.Element => {
+  const {
+    loading,
+    isAuthenticated,
+    getTokenSilently,
+    loginWithRedirect,
+    logout,
+    user
+  } = useAuth0();
+
+  useEffect(() => {
+    if (loading) {
+      return;
+    } else if (!isAuthenticated) {
+      loginWithRedirect({});
+    }
+
+    const setToken: () => void = async (): Promise<void> => {
+      const token: string = (await getTokenSilently()) || "";
+      localStorage.setItem(config.ACCESS_TOKEN_KEY, token);
+    };
+
+    setToken();
+  }, [loading, isAuthenticated, getTokenSilently]);
+
+  const onClick: () => void = (): void => {
+    logout({ returnTo: config.PAGE_URI });
+  };
+
+  if (loading || !isAuthenticated) {
+    return <Spin size="large" className="load-page" tip="Loading..." />;
+  }
+
+  const func: () => undefined = (): undefined => undefined;
```

Comment:
Does sider require an onBreakpoint function? If no, can we remove this.

It may also make more sense to return `void` even thought the linter may not like it